reviews/2022-cert-manager-annual.md

```diff
@@ -0,0 +1,93 @@
+
+The project has remained a prominent tool in the Cloud Native space since its inclusion in the CNCF sandbox, fuelled by wide community adoption and strong ambition to keep on growing and improving in the future.
+
+_NB: Statistics referring to the last year mean "the year up to 2021-12-02"_
+
+## DevStats
+
+[cert-manager's DevStats page](https://certmanager.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m&from=now-1y&to=now-1h) shows a steady rate of contributions throughout the last year.
+
+Importantly for the project, we saw at least [45 new first-time-contributors](https://certmanager.devstats.cncf.io/d/52/new-contributors-table?orgId=1&from=now-1y&to=now&var-repogroup_name=cert-manager&viewPanel=1), only counting the main cert-manager repository.
+
+Among those brand new contributors, cert-manager added 3 official [maintainers](https://github.com/jetstack/cert-manager/blob/71de30931fc54eba59853097b1699ffdc4bbc974/OWNERS) and two contributors who took part in Google Summer of Code 2021: [Arsh Sharma](https://summerofcode.withgoogle.com/archive/2021/projects/4983164438577152/) ([RinkiyaKeDad](https://github.com/RinkiyaKeDad)) and [Tim Ramlot](https://summerofcode.withgoogle.com/archive/2021/projects/5686178510012416/) ([inteon](https://github.com/inteon)). In addition, a previous contributor joined the maintainer team.
+
+The source of commits is varied. A large percentage of activity at any given time is usually from the core maintainers, most of whom work at Jetstack. That said, there are regular contributions from a wide [variety of companies](https://certmanager.devstats.cncf.io/d/4/company-statistics-by-repository-group?orgId=1&var-period=d7&var-metric=activity&var-repogroup_name=cert-manager&var-companies=All&from=now-1y&to=now).
+
+In addition, the time taken for a contribution to get merged has largely [remained steady](https://certmanager.devstats.cncf.io/d/16/opened-to-merged?orgId=1&var-period=m&var-repogroup_name=cert-manager&from=now-1y&to=now).
+
+## Maintainers
+
+As mentioned under DevStats, the project has added several new maintainers over the past year.
+
+[Irbe Krumina (irbekrm)](https://github.com/irbekrm), [Ashley Davis (SgtCoDFish)](https://github.com/SgtCoDFish), [Joakim Ahrlin (jahrlin)](https://github.com/jahrlin) and [Maël Valais (maelvls)](https://github.com/maelvls) - all at Jetstack - joined the [maintainers](https://github.com/jetstack/cert-manager/blob/71de30931fc54eba59853097b1699ffdc4bbc974/OWNERS) team, bringing the total number of maintainers to 9 with one maintainer at Apple, seven at Jetstack and one at Thomas More University in Belgium.
+
+## Adoption
```

Comment:
Thought: perhaps not exactly pointing to adoption as such, but cert-manager is also included as part of installation or recommended as a means to obtain TLS certs by various other widely used projects i.e [cluster api](https://cluster-api.sigs.k8s.io/developer/guide.html#cert-manager), [knative](https://knative.dev/development/install/serving/installing-cert-manager/), [istio](https://istio.io/latest/docs/ops/integrations/certmanager/). I wonder if it'd be worth pointing out that and that cert-manager aims to be seen almost as part of a default 'control-plane' setup for any cluster

Reply:
Brilliant suggestions, I'll add them in straight away. Thanks :grin: